### PR TITLE
docs: improved the zh version of create deployment in reference/sql/deployment_manage

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
fixed links 
fixed typos in create table
improved the structure and synatx introduction

### CODE REVIEW

File: docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 1]
```diff
@@ -98,52 +103,55 @@ LongWindowOption
...
      - 支持的 where op: `>, <, >=, <=, =, !=`
 
      - where 关联的列 `<column ref>`，数据类型不能是 date 或者 timestamp
 
+#### Example
+```sql
+DEPLOY demo_deploy OPTIONS(long_windows="w1:1d") SELECT c1, sum(c2) OVER w1 FROM demo_table1
+    WINDOW w1 AS (PARTITION BY c1 ORDER BY c2 ROWS_RANGE BETWEEN 5d PRECEDING AND CURRENT ROW);
+
+I0908 08:06:15.512660  1091 zk_client.cc:58] node watcher with event type 3, state 3
```
Comment (vagetablechicken) [inline docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: There're stdout logs, will removed later, it's better to delete these.

Comment (michelle-qinqin) [reply docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: logs in both examples have been removed

**Review by vagetablechicken — APPROVED** [review]

File: docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 2]
```diff
@@ -4,27 +4,29 @@
...
+`DeployOption`的定义详见[DEPLOYMENT属性DeployOption（可选）](#DeployOption可选)。
+`SelectStmt`的定义详见[Select查询语句](../dql/SELECT_STATEMENT.md)。
+`DEPLOY`语句可以将SQL部署到线上。OpenMLDB仅支持部署Select查询语句，并且需要满足[OpenMLDB SQL上线规范和要求](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.md)。
 
-```SQL
-DEPLOY deployment_name SELECT clause
-```
 
-### Example: 部署一个SQL到online serving
+#### Example
```
Comment (lumianph) [inline docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: Example 尽量不要把他作为一个单独的section，不然目录里面呈现会比较奇怪，你可以黑体加粗，不加粗也没关系

Comment (lumianph) [reply docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: 下面还有一个example作为了单独section

**Review by lumianph — APPROVED** [review]

**Pull request merged**